dart/lib/flat_buffers.dart

```diff
@@ -712,7 +717,7 @@ class Builder {
     int alignDelta = (-(_tail + dataSize)) % size;
     int bufSize = alignDelta + dataSize;
     // Ensure that we have the required amount of space.
-    {
+        {
```

Comment:
Revert new whitespace

Reply:
hmm. can I maybe apply `dartfmt -w .` at the very end after this PR is reviewed and accepted?